### PR TITLE
ZEB-1264/ fix change status button

### PR DESCRIPTION
**Pull request by MitskevichVS**
Fixed: status button (disable) + link issue button

### CODE REVIEW

File: client/app/containers/test-run-info/test-run-info.html [hunk 1]
```diff
@@ -98,20 +98,23 @@
                                                                     </td>
                                                                     <td class="tab-additional__table-data _status-info _border-bottom">
                                                                         <md-menu md-offset="30 20" class="tab-additional__menu">
-                                                                            <div class="menu__button" aria-label="Additional menu" ng-click="$mdMenu.open($event)">
+                                                                            <md-button class="menu__button"
+                                                                                aria-label="Additional menu"
+                                                                                ng-click="$mdMenu.open($event)"
+                                                                                ng-disabled="$ctrl.test.status.split('_').join(' ').toLowerCase() !== 'passed' && $ctrl.test.status.split('_').join(' ').toLowerCase() !== 'failed'">
```
Comment (danymill) [inline client/app/containers/test-run-info/test-run-info.html:104]: `.split('_').join(' ')` is redundant

File: client/app/containers/test-run-info/test-run-info.html [hunk 2]
```diff
@@ -98,20 +98,23 @@
...
                                                                                 <span class="button-text _{{$ctrl.test.status.split('_').join(' ').toLowerCase()}}">{{ $ctrl.test.status.split('_').join(' ').toLowerCase() }}</span>
-                                                                                <span class="material-icons button-icon">arrow_drop_down</span>
-                                                                            </div>
+                                                                                <span class="material-icons button-icon" ng-show="$ctrl.test.status.split('_').join(' ').toLowerCase() === 'passed' || $ctrl.test.status.split('_').join(' ').toLowerCase() === 'failed'">arrow_drop_down</span>
```
Comment (danymill) [inline client/app/containers/test-run-info/test-run-info.html:106]: the same about `.split('_').join(' ')`

File: client/app/containers/test-run-info/test-run-info.html [hunk 3]
```diff
@@ -98,20 +98,23 @@
...
+                                                                                    <md-button ng-click="$ctrl.changeTestStatus($ctrl.test, item)">
                                                                                         <span class="item-text {{item}}" md-menu-align-target>{{item}}</span>
                                                                                     </md-button>
                                                                                 </md-menu-item>
                                                                             </md-menu-content>
                                                                         </md-menu>
                                                                     </td>
-                                                                    <td class="tab-additional__table-data _issue _border-bottom">
+                                                                    <td class="tab-additional__table-data _issue _border-bottom" ng-if="$ctrl.test.status.toLowerCase() === 'failed'">
```
Comment (danymill) [inline client/app/containers/test-run-info/test-run-info.html:117]: The same about `.split('_').join(' ')`

**Review by danymill — CHANGES_REQUESTED** [review]

File: client/app/containers/test-run-info/test-run-info.html (hunk 3 above)
Comment (MitskevichVS) [reply client/app/containers/test-run-info/test-run-info.html:117]: There is no split().join() )

File: client/app/containers/test-run-info/test-run-info.html (hunk 2 above)
Comment (MitskevichVS) [reply client/app/containers/test-run-info/test-run-info.html]: fixed

File: client/app/containers/test-run-info/test-run-info.html (hunk 1 above)
Comment (MitskevichVS) [reply client/app/containers/test-run-info/test-run-info.html]: fixed

**Review by danymill — APPROVED** [review]

**Pull request merged**